tests/plugins/plugins.spec.js

```diff
@@ -0,0 +1,66 @@
+const { mainTest } = require('../../fixtures');
+const { MainPage } = require('../../pages/workspace/main-page');
+const { expect, test } = require('@playwright/test');
+const { random } = require('../../helpers/string-generator');
+const { TeamPage } = require('../../pages/dashboard/team-page');
+const { DashboardPage } = require('../../pages/dashboard/dashboard-page');
+const { PluginsPage } = require('../../pages/workspace/plugins-page');
+const { qase } = require('playwright-qase-reporter/dist/playwright');
+
+const teamName = random().concat('autotest');
+
+test.beforeEach(async ({ page }) => {
+  const teamPage = new TeamPage(page);
+  const dashboardPage = new DashboardPage(page);
+  const mainPage = new MainPage(page);
+  await teamPage.createTeam(teamName);
+  await teamPage.isTeamSelected(teamName);
+  await dashboardPage.createFileViaPlaceholder();
+  await mainPage.isMainPageLoaded();
+});
+
+test.afterEach(async ({ page }, testInfo) => {
+  const teamPage = new TeamPage(page);
+  const mainPage = new MainPage(page);
+});
+
+mainTest.slow();
+
+mainTest(qase([1837,1838, 1839, 1844], 'Install a plugin by URL (via toolbar), close,  Plugin Manager',), async ({ page }) => {
+  const mainPage = new MainPage(page);
+  const pluginsPage = new PluginsPage(page);
+  const LOREM_IPSUM_PLUGIN_URL = 'https://lorem-ipsum-penpot-plugin.pages.dev/assets/manifest.json';
+  
+  await mainPage.clickPluginsButton();
+  await expect(pluginsPage.pluginPanel).toBeVisible();
+  
+  // 1837, Install a plugin by URL (via plugin icon in toolbar)
+  await pluginsPage.setPluginUrl(LOREM_IPSUM_PLUGIN_URL);
+  await pluginsPage.clickOnInstallPluginButton();
+  await pluginsPage.clickOnAllowPluginButton();
+  await pluginsPage.checkInstalledPluginsCountIs(1);
+  await expect(pluginsPage.pluginPanel).toHaveScreenshot(
+    'plugin-manager--plugin-installed.png'
+  );
+
+  // 1838, Close the "Plugins Manager" modal
+  await pluginsPage.clickClosePluginPanelButton();
+
+  // 1844, Open a plugin (via Main menu 3 dots > Plugins > plugin name)
+  await mainPage.clickMainMenuButton();
+  await mainPage.clickPluginsMainMenuItem();
+  await mainPage.clickLoremIpsumButton();
+  await expect(mainPage.loremPluginPanel).toHaveScreenshot(
+    'lorem-ipsum-plugin-panel.png'
+  );
+
+  // 1839, Delete a plugin from the "Plugins Manager" modal (via delete icon button)
+  await mainPage.clickMainMenuButton();
+  await mainPage.clickPluginsMainMenuItem();
+  await mainPage.clickPluginsManagerButton();
+  await pluginsPage.clickOnUninstallPluginButton();
+  await pluginsPage.checkInstalledPluginsCountIs(0);
+  await expect(pluginsPage.pluginPanel).toHaveScreenshot(
+    'plugin-manager--no-plugins-installed.png'
```

Comment:
You have two -- in the file's name. Is it a mistake?

Reply:
It was intended, but now I find it better to use a single dash. I'll rename that and the snapshots pictures.